controllers/cartController.js

```diff
@@ -25,6 +25,55 @@ const createCart = catchAsync(async (req, res) => {
   }
 });
 
+const getCart = catchAsync(async (req, res) => {
+  try {
+    const userId = req.user.id;
+
+    const carts = await cartService.getCart(userId);
+
+    res.status(201).json({ carts });
+  } catch (error) {
+    console.log(error);
+    res.status(error.statusCode).json({ message: error.message });
+  }
+});
+
+const updateCart = catchAsync(async (req, res) => {
+  try {
+    const userId = req.user.id;
+    const { cartId } = req.params;
+    const { quantity } = req.body;
+
+    if (!userId || !cartId || !quantity) {
+      return res.status(400).json({ message: "KEY_ERROR" });
+    }
+
+    const updateCart = await cartService.updateCart(userId, cartId, quantity);
+
+    res.status(200).json({ updateCart });
+  } catch (error) {
+    console.log(error);
+    res.status(error.statusCode).json({ message: error.message });
+  }
+});
+
+const deleteCart = catchAsync(async (req, res) => {
+  try {
+    const userId = req.user.id;
+    const { cartId } = req.params;
+
+    await cartService.deleteCart(userId, cartId);
+
+    res.status(204).json({ message: "delete Cart " });
```

Comment:
```suggestion
    res.status(204).json();
```
`204` statusCode는 아마 body에 담아도 전달되는 값이 없을 것 같은데 한번 확인해보시겠어요?